crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -116,6 +119,7 @@ def adminrepocheck
     end
   rescue Crowbar::Error::UpgradeError,
          StandardError => e
+    log_exception(e)
```

Comment:
it seems better to split the handling of `StandardError` and `Crowbar::Error::UpgradeError`